src/apps/loan-list/list/loan-list.tsx

```diff
@@ -82,39 +80,56 @@ const LoanList: FC<LoanListProps> = ({ pageSize }) => {
   return (
     <div className="loan-list-page">
       <h1 className="text-header-h1 my-32">{t("loanListTitleText")}</h1>
-      {(physicalLoans.length > 0 || digitalLoans.length > 0) && (
+      {((Array.isArray(physicalLoans) && physicalLoans.length > 0) ||
+        (Array.isArray(digitalLoans) && digitalLoans.length > 0)) && (
         <>
           {physicalLoans && (
             <List
               pageSize={pageSize}
               emptyListLabel={t("loanListPhysicalLoansEmptyListText")}
-              header={t("loanListPhysicalLoansTitleText")}
               loans={physicalLoans}
               dueDates={physicalLoansDueDates}
-              setView={setView}
-              view={view as ListView}
-              viewToggleable
-            />
+              view={view}
+            >
+              <ListHeader
+                header={t("loanListPhysicalLoansTitleText")}
+                amount={physicalLoans.length}
+              >
+                <ToggleListViewButtons
+                  disableRenewLoansButton={
+                    getAmountOfRenewableLoans(physicalLoans) === 0
+                  }
+                  view={view}
+                  setView={setView}
+                  loans={physicalLoans}
+                  pageSize={pageSize}
+                />
+              </ListHeader>
+            </List>
           )}
           {digitalLoans && (
             <List
               pageSize={pageSize}
-              header={t("loanListDigitalLoansTitleText")}
               emptyListLabel={t("loanListDigitalLoansEmptyListText")}
               loans={digitalLoans}
-              dueDates={digitalLoansDueDates}
-              setView={setView}
-              view={view as ListView}
-              viewToggleable={false}
-            />
+              view="list"
+            >
+              <ListHeader
+                header={t("loanListDigitalLoansTitleText")}
+                amount={digitalLoans.length}
+              />
+            </List>
           )}
         </>
       )}
-      {physicalLoans.length === 0 && digitalLoans.length === 0 && (
-        <EmptyList
-          emptyListText={t("loanListDigitalPhysicalLoansEmptyListText")}
-        />
-      )}
+      {Array.isArray(physicalLoans) &&
+        physicalLoans.length === 0 &&
+        Array.isArray(digitalLoans) &&
+        digitalLoans.length === 0 && (
```

Comment:
You could create a helper function here or I think lodash have an utility function for it too:
```typescript
const loansAreEmpty = (loans: something[]) => Array.isArray(loans) && !loans.length
```